CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# [1.0.0](https://github.com/grpc-ecosystem/grpc-gateway/compare/v1.8.5...v1.0.0) (2019-03-28)
+
+
+
+# [1.0.0](https://github.com/grpc-ecosystem/grpc-gateway/compare/v1.8.5...v1.0.0) (2019-03-28)
+
+
+
+# [1.0.0](https://github.com/grpc-ecosystem/grpc-gateway/compare/v1.8.5...v1.0.0) (2019-03-28)
+
+
+
+# [1.0.0](https://github.com/grpc-ecosystem/grpc-gateway/compare/v1.8.5...v1.0.0) (2019-03-28)
+
+
+
```

Comment:
Remove these please

Reply:
roger

Reply:
It is automatically generated by the configured commit hook. You edit package.json for styles you want for the team.

Reply:
Can you please remove anything that isn't directly related to the documentation change? Including the node module changes, package.json, etc. We can discuss new commit hooks separately, not as part of this change.

Reply:
Roger. You are right that we can discuss commit hook in a different thread.